environment/sql/data.sql

```diff
@@ -23,6 +23,7 @@ INSERT INTO `owner` (`id`) VALUES
 (442711068737929216);
 
 INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES
+<<<<<<< HEAD
```

Comment:
You accidentally left in a conflict marker

Reply:
jsut take them out merging